pkg/boot/multiboot/multiboot.go

```diff
@@ -313,6 +349,41 @@ func (m *multiboot) addInfo() (addr uintptr, err error) {
 	return r.Start, nil
 }
 
+func (*mutibootHeader) addInfo(m *multiboot) (addr uintptr, err error) {
```

Comment:
I think most of the other functions in this file have comments, could you add one for this too?